tests/unit/test_allowlist.py

```diff
@@ -46,39 +46,151 @@ def allowlist():
+            id=7,
+            namespace="gitlab.com/packit-service/src",
+            status=AllowlistStatus.approved_automatically.value,
+        ),
+        "github.com/packit/packit.git": flexmock(
+            id=9,
+            namespace="github.com/packit/packit.git",
+            status=AllowlistStatus.approved_manually.value,
+        ),
+        "gitlab.com/packit/packit.git": flexmock(
+            id=11,
+            namespace="gitlab.com/packit/packit.git",
+            status=AllowlistStatus.denied.value,
+        ),
+        "github.com/packit/denied_packit.git": flexmock(
+            id=12,
+            namespace="github.com/packit/denied_packit.git",
+            status=AllowlistStatus.denied.value,
+        ),
+        "github.com/fero/denied_packit.git": flexmock(
+            id=13,
+            namespace="github.com/fero/denied_packit.git",
+            status=AllowlistStatus.denied.value,
+        ),
+    }
+
+    return entries
+
+
+def mock_model(entries, namespaces):
+    for queried_namespace in namespaces:
+        (
+            flexmock(DBAllowlist)
+            .should_receive("get_namespace")
+            .with_args(queried_namespace)
+            .and_return(entries.get(queried_namespace))
+            .ordered()
+        )
+
+
+@pytest.fixture()
+def mocked_model(allowlist_entries, request):
+    mock_model(allowlist_entries, request.param)
+
+
 @pytest.mark.parametrize(
-    "account_name, model, is_approved",
+    "account_name, mocked_model, is_approved",
     (
-        (
-            "fero",
-            flexmock(
-                id=1,
-                account_name="fero",
-                status=AllowlistStatus.approved_manually.value,
-            ),
+        (  # Fero is approved
+            "github.com/fero",
+            ("github.com/fero",),
             True,
         ),
-        (
-            "lojzo",
-            flexmock(
-                id=2,
-                account_name="lojzo",
-                status=AllowlistStatus.approved_automatically.value,
-            ),
+        (  # Lojzo is approved
+            "gitlab.com/lojzo",
+            ("gitlab.com/lojzo",),
             True,
         ),
-        (
-            "konipas",
-            flexmock(
-                id=3, account_name="konipas", status=AllowlistStatus.waiting.value
+        (  # Lojzo is approved on GitLab, shall not pass on GitHub
+            "github.com/lojzo",
+            ("github.com/lojzo", "github.com"),
+            False,
+        ),
+        (  # Konipas is waiting... checks parent if approved(???)
+            "github.com/konipas",
+            ("github.com/konipas", "github.com"),
+            False,
+        ),
+        (  # Krasomila is not present at all, checks parent
+            "github.com/krasomila",
+            ("github.com/krasomila", "github.com"),
+            False,
+        ),
+        (  # gitlab.com/packit-service/src is approved
+            "gitlab.com/packit-service/src/glibc.git",
```

Comment:
oh, you're checking it here

Reply:
oh, I interpreted it as another subnamespace in the comment above

Reply:
I just wanted to be sure the behaviour is correct above